File: integrations/java/app/revanced/integrations/youtube/patches/ZoomHapticsPatch.java
package app.revanced.integrations.youtube.patches;

import app.revanced.integrations.youtube.settings.Settings;

@SuppressWarnings("unused")
public class ZoomHapticsPatch {
    public static boolean shouldVibrate() {
        return !Settings.DISABLE_ZOOM_HAPTICS.get();
    }
}

File: patches/src/main/kotlin/app/revanced/patches/youtube/misc/zoomhaptics/ZoomHapticsPatch.kt
package app.revanced.patches.youtube.misc.zoomhaptics

import app.revanced.patcher.patch.bytecodePatch
import app.revanced.patches.youtube.misc.hapticfeedback.disableHapticFeedbackPatch

@Deprecated("Superseded by disableHapticFeedbackPatch", ReplaceWith("disableHapticFeedbackPatch"))
val zoomHapticsPatch = bytecodePatch(
    description = "Adds an option to disable haptics when zooming.",
) {
    dependsOn(disableHapticFeedbackPatch)
}